### PR TITLE
logback documentation

### PR DESCRIPTION
**Pull request by poi33**
Giving the logback documetation an attempt since no one want to document it.

### CODE REVIEW

File: docs/deployment/config.adoc [hunk 1]
```diff
@@ -535,6 +535,75 @@ loginWithoutUser = true
 
 loginWithoutUser:: Set to false to force creation of user before logging in. Default: `true`.
 
+=== Logback
+
+`XP_HOME/config/logback.xml`
+
+Configure all logging inside the system.
+
+.Default logback file
```
Comment (rymsha) [inline docs/deployment/config.adoc]: Well, it is not true since 7.2, I think.
Also more changes made in master

Comment (poi33) [reply docs/deployment/config.adoc]: Do we have new default file?

Comment (rymsha) [reply docs/deployment/config.adoc]:
```
<?xml version="1.0"?>
<configuration scan="true" scanPeriod="60 seconds">

  <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
    <file>${xp.home}/logs/server.log</file>
    <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
      <fileNamePattern>${xp.home}/logs/server.%d{yyyy-MM-dd}.%i.log</fileNamePattern>
      <maxFileSize>100MB</maxFileSize>
      <maxHistory>7</maxHistory>
      <totalSizeCap>3GB</totalSizeCap>
    </rollingPolicy>
    <encoder>
      <pattern>%d{HH:mm:ss.SSS} %-5level %logger{36} - %msg%n</pattern>
    </encoder>
  </appender>

  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
    <withJansi>true</withJansi>
    <encoder>
      <pattern>%date{ISO8601} %highlight(%-5level) %cyan(%logger{36}) - %msg%n</pattern>
    </encoder>
  </appender>

  <root level="info">
    <appender-ref ref="STDOUT"/>
    <appender-ref ref="FILE"/>
  </root>

  <logger name="Events.Service" level="WARN" additivity="false">
    <appender-ref ref="STDOUT"/>
    <appender-ref ref="FILE"/>
  </logger>

  <logger name="Events.Bundle" level="WARN" additivity="false">
    <appender-ref ref="STDOUT"/>
    <appender-ref ref="FILE"/>
  </logger>

</configuration>

```

Comment (poi33) [reply docs/deployment/config.adoc]: Then it should be included in the newest commit.

**Pull request merged**